drivers/sensor/stm32_temp/stm32_temp.c

```diff
@@ -67,13 +70,24 @@ static int stm32_temp_channel_get(const struct device *dev,
 		return -ENOTSUP;
 	}
 
+#ifdef CONFIG_SOC_SERIES_STM32L5X
+	temp = ((float)data->raw * cfg->tsv_mv) / TEMPSENSOR_CAL_VREFANALOG;
```

Comment:
do you want to assert for a non-zero value of `TEMPSENSOR_CAL_VREFANALOG`

Reply:
done